tests/test_review_dc_people.py

```diff
@@ -0,0 +1,54 @@
+"""
+
+url = "https://doctrineandcovenantscentral.org/people-of-the-dc/almon-whiting-babbitt/"
+
+
+def check_text(url: str, markdown_content: str) -> Any:
+    """Check for unwanted text presence"""
+    # result check variable boolean
+    result_check = []
+
+    soup = BeautifulSoup(markdown_content, "html.parser")
+    markdown_content = soup.prettify()
+
+    doc_content = load_dc_people(url, markdown_content)
+
+    # Search for the position of "unwanted text" (case insensitive)
+    text_match = re.search(r"[(\d+)](#t(\d+)).", doc_content.page_content, re.IGNORECASE)
+
+    if text_match:
+        # Check content after the "text" text
+        text_position = text_match.start()
+
+        # Check if there is text after the "text" section
+        if text_position > 0:
+            result_check.append("True")
+        else:
+            result_check.append("False")
+
+    return result_check
+
+
+def test_review_load_dc_people() -> None:
+    """It returns a valid Document for load dc people."""
```

Comment:
Again, this test file should contain HTML. It should not define a check_text function. It should import load_dc_people from the load_dc_people.py file, pass the HTML into that function, and then assert that `[1](#t1).` does not appear in the resulting doc.page_content.